user-console/src/main/java/org/pentaho/mantle/client/commands/DeleteFolderCommand.java

```diff
@@ -151,7 +172,9 @@ public void onResponseReceived( Request request, Response response ) {
     folderDeleteWarningDialogBox.center();
 
   }
-
+  public static native boolean isActiveElement( Element e )/*-{
+    return $doc.activeElement === e;
+ }-*/;
```

Comment:
Not needed anymore, but, btw, already exists as `ElementUtils.isActiveElement(.)`.